markdown/treeprocessors.py

```diff
@@ -45,7 +46,7 @@ def build_treeprocessors(md: Markdown, **kwargs: Any) -> util.Registry[Treeproce
     return treeprocessors
 
 
-def isString(s: Any) -> bool:
+def isString(s: object) -> TypeGuard[str]:
```

Comment:
Groan. Really? Okay, maybe it is more accurate with the change, but who cares? Python doesn't and so why should I? Treating the return value as a bool is all I need to know. Knowing the return type is `TypeGaurd` provides no additional value.

I'm not going to comment on every instance. This is just one glaring example. And I'm not going to demand that the change be reverted. But this expresses my thoughts on type annotations in Python in general.

Reply:
This makes it as good as an `isinstance` check for purposes of static typing. I can just drop it, just to make this PR uncontroversial.